src/config/ratelimit.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I'm glad you're not as lazy as me, I probably would have just put these in with the other configs, but this is cleaner